.github/workflows/test-unit.yml

```diff
@@ -31,3 +31,8 @@ jobs:
 
       - name: Run unit tests
         run: make test-unit
```

Comment:
Do we want to change our `test-unit` make target so that it generates a code coverage report to upload?

<img width="1284" alt="image" src="https://github.com/defenseunicorns/zarf/assets/87675701/b7cc9c6c-68f6-41c1-9885-e071cd448f01">

https://github.com/defenseunicorns/zarf/actions/runs/9178093280/job/25237190251?pr=2529#step:5:44

https://github.com/defenseunicorns/zarf/blob/29d3ee39dfcbe173405f6e0010cd0dfb9468cd6e/Makefile#L209-L213

Reply:
I don't think it needs to be in the make target, but I did add the line we talked about earlier.

Reply:
Nice! We should be able to remove make test-unit from this workflow, because now this workflow runs the unit tests twice